contrib/ansible/tasks/bootstrap_ubuntu.yaml

```diff
@@ -7,6 +7,6 @@
       - unzip
       - tar
       - apt-transport-https
-      - btrfs-tools
+      - libbtrfs-dev
```

Comment:
libbtrfs-dev is no longer needed with this:
- https://github.com/containerd/containerd/pull/7933